### ISSUE TITLE
BCAS20071 v1.00

### ISSUE THREAD

**Issue by Ponsen** [gibbed/DemonsSoulsDebug#6]
Hi,
I have a Version which doesn't seem to be supported by the current releases.
RPCS says it's BCAS20071 v1.00 and the EBOOT.elf has the following checksum 
actual: e5b0469148f4217a540f1bae33bbd2bf63648c2d

Any chance to get the patch working for my version?

Thanks a lot in advance!

**Comment by gibbed** [comment]
Is there a reason you don't want to patch to v1.04?

**Comment by Ponsen** [comment]
Yes, i'd like to try some speed running and some things don't work in the patched versions.

**Comment by gibbed** [comment]
Alright, I'll put this on my list but I don't think I'll get to it any time soon. I'd suggest checking if the patched executable can run with unpatched game data and if the things you're attempting to do would work then.

**Comment by Ponsen** [comment]
Thansk a lot.
I've patched the game, but it doesn't seem to work.
I tried placing the debug.elf to the game and update folders (alongside the EBOOT.BIN) but no debug menu ingame or main screen.
Replacing debug.elf with EBOOT.BIN results in a white screen, the game doesn't seem to load.
Loading the debug.elf from the game/update directory directly via RPCS loads up the game, but also no menu.

Any suggestions?

**Comment by gibbed** [comment]
Debug menu is opened by pressing select, unless you mean you're not seeing the debug title menu either. What I do when emulating is drop `debug.elf` next to `EBOOT.BIN` and use `File > Boot SELF/ELF` in the menu.

**Comment by Ponsen** [comment]
I'm pressing the "Back" Button on my Xbox controller, which is mapped to Select, I checked.
As described I'm doing the same steps to load the debug.elf, but the debug-menu doesn't show up. Not in the title screen or ingame.
